charts/enterprise-operator/README.md

```diff
@@ -0,0 +1,71 @@
+# MongoDB Enterprise Kubernetes Operator Helm Chart
+
+A Helm Chart for installing and upgrading the [MongoDB Enterprise
+Kubernetes Operator](https://github.com/mongodb/mongodb-enterprise-kubernetes).
+
+## Prerequisites
+
+The installation of this Chart does not have prerequisites. However, in order to
+create a Mongo Database in your Kubernetes cluster, you'll need a [Cloud
+Manager](https://cloud.mongodb.com) account.
+
+## Installing Enterprise Operator
+
+You can install the MongoDB Enterprise Operator easily with:
+
+``` shell
+helm install enterprise-operator mongodb/enterprise-operator
+```
+
+This will install `CRD`s and the Enterprise Operator in the current namespace
+(`default` by _default_). You can pass a different namespace with:
+
+``` shell
+helm install enterprise-operator mongodb/enterprise-operator --namespace mongodb [--create-namespace]
+```
+
+To install the Enterprise Operator in a namespace called `mongodb`; with the
+optional `--create-namespace` Helm will create the Namespace if it does not exist.
+
+## Configuring access to Cloud Manager
+
+The Enteprise Operator can run against an account of [Cloud
+Manager](https://cloud.mongodb.com) or [Ops
+Manager](https://www.mongodb.com/products/ops-manager).
+
+### Using Cloud Manager
+
+Please visit "[Configure Kubernetes for Deploying MongoDB
+Resource](https://docs.cloudmanager.mongodb.com/tutorial/nav/k8s-config-for-mdb-resource/)"
+in order to create new `Secret`s and `ConfigMap`s with credentials for your
+Cloud Manager project.
+
+You will create a `ConfigMap` with name `my-project`, and a `Secret` with name
+`my-credentials`.
+
+## Deploying a MongoDB Replica Set
```

Comment:
We need to add the chart for an enterprise operator replica set (with enterprise backing db)
Then also we should have some comments on how to use community operator with the enterprise dbs.

Reply:
Yes, we have time allocated for that and will do as a follow-up to this patch.